app/src/main/java/com/keylesspalace/tusky/MainActivity.java

```diff
@@ -593,4 +596,20 @@ public FloatingActionButton getActionButton() {
         return fragmentInjector;
     }
 
+    @Override
+    public boolean dispatchKeyEvent(KeyEvent event) {
```

Comment:
There is already a `onKeyDown` method in `MainActivity` where we handle other shortcuts. Not sure where the right place is for this, but it should be one single place to not clutter the code.